config/v1/types_node.go

```diff
@@ -77,6 +77,7 @@ type NodeStatus struct {
 }
 
 // +kubebuilder:validation:Enum=v1;v2;""
+// +kubebuilder:validation:XValidation:rule="self != \"v1\"",message="cgroups v1 is not supported on openshift anymore"
```

Comment:
Why not just remove the value from the enum?

Reply:
Would that change be enough? I thought removal of enum might end up in an error
cc: @rphillips

Reply:
If you can prove it won't using ratcheting, that's the simplest change